friends/src/components/Form.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const Form = props => {
+
+
+    return (
+        <div className="form">
+            <form onSubmit={props.createFriend}>
+            <input placeholder="name..." name="name" onChange={props.handleChanges}></input>
+            <input placeholder="age..." name="age" onChange={props.handleChanges}></input>
+            <input placeholder="email..." name="email" onChange={props.handleChanges}></input>
+            </form>
+            <button onClick={props.createFriend} >Add New Friend</button>
```

Comment:
Update the component to include the changes we made in standup such as placing the button inside the form and changing the button type to submit.